custom_components/powercalc/data/tp-link/KL135/model.json

```
@@ -0,0 +1,13 @@
+{
+    "calculation_strategy": "lut",
+    "measure_description": "Measured with utils/measure script",
+    "measure_device": "TP-Link Kasa KP115",
+    "measure_method": "script",
+    "measure_settings": {
+        "SAMPLE_COUNT": 2,
+        "SLEEP_TIME": 3,
+        "VERSION": "v1.3.0:docker"
+    },
+    "name": "TP-Link Kasa KL135(US)",
+    "standby_power": 0.05
```

0.05 is too low for standby power.

According to the manufacturer specs it should be somewhere around 0.2 Watt, so I suggest to set it to that value if you are not able to measure it (because it's too low for the smart plug to pick up).

```suggestion
    "standby_power": 0.2
```

See https://www.kasasmart.com/us/products/smart-lighting/kasa-smart-light-bulb-multicolor-kl135
Compared with other smart bulbs whose standby powers are more than 0.5W: when set in standby mode, a Kasa smart bulb's exact power consumption is as low as 0.2 watts.